packages/hadron-document/src/element.js

```diff
@@ -41,7 +41,8 @@ const UNEDITABLE_TYPES = [
   'Timestamp',
   'BSONRegExp',
   'Undefined',
-  'Null'
+  'Null',
+  'DBRef'
```

Comment:
Editing for this type doesn't work. We can make it work, but for now I'm just disabling it similar to some other types here